pkgs/by-name/ko/kolla/package.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, python311
+, fetchgit
+, bashate
+}:
+
+let
+  pythonPackages = python311.pkgs;
+
+in
+pythonPackages.buildPythonPackage rec {
```

```suggestion
, python311Packages
, fetchgit
, bashate
}:

python311Packages.buildPythonApplication rec {
```

`buildPythonPackage` is for libraries, you'll probably want this. Also, you can consume `python311Packages` directly.